plugins/org.locationtech.udig.project.tests/src/org/locationtech/udig/project/internal/render/impl/renderercreator/SingleFeatureRenderer2.java

```diff
@@ -0,0 +1,33 @@
+/* uDig - User Friendly Desktop Internet GIS client
+ * http://udig.refractions.net
+ * (C) 2004, Refractions Research Inc.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * (http://www.eclipse.org/legal/epl-v10.html), and the Refractions BSD
+ * License v1.0 (http://udig.refractions.net/files/bsd3-v10.html).
+ */
+package org.locationtech.udig.project.internal.render.impl.renderercreator;
+
+import java.awt.Graphics2D;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.locationtech.udig.project.internal.render.impl.RendererImpl;
+import org.locationtech.udig.project.render.RenderException;
+
+/**
+ * Simulates a simple renderer that renders a single class.
+ * 
+ * @author Nikolaos Pringouris <nprigour@gmail.com>
+ */
+public class SingleFeatureRenderer2 extends RendererImpl {
```

Comment:
Same here, ist it to have two different type of RendererImpl? What about using the same class twice for two different renderer-Extensions in plugin.xml?

Reply:
Actually it will make easier to distinguish the preferred renderer during testing via the extra menu option provided since class names are used for display and select 
![image](https://user-images.githubusercontent.com/26147785/40113560-8cec9456-5912-11e8-8fc4-b56a31bae9f9.png)